source/StoryMenuState.hx

```diff
@@ -396,4 +410,27 @@ class StoryMenuState extends MusicBeatState
 		intendedScore = Highscore.getWeekScore(curWeek, curDifficulty);
 		#end
 	}
+
+	public static function unlockNextWeek(week:Int):Void
+	{
+		// TODO: get the weekData from class (but making it static makes it dissapear when beating the week)
```

Comment:
I need someone to look at this for me,
When `weekData` is static and you beat fro example week 1 `weekData[1]` gets nulled